prow/cmd/pipeline/controller.go

```diff
@@ -317,6 +331,18 @@ func (c *controller) deletePipelineRun(pContext, namespace, name string) error {
 	return p.client.TektonV1beta1().PipelineRuns(namespace).Delete(context.TODO(), name, metav1.DeleteOptions{})
 }
 
+func (c *controller) cancelPipelineRun(pContext string, pr *pipelinev1beta1.PipelineRun) (*pipelinev1beta1.PipelineRun, error) {
```

Comment:
I read `pr` and think Pull Request, can we use a different abbreviation? :P

Reply:
I used the existing abbreviation but I can change it as well, no problem.